tephra-core/src/main/java/org/apache/tephra/TxConstants.java

```diff
@@ -345,4 +345,14 @@
     public static final byte CURRENT_VERSION = 3;
   }
 
+  /**
+   * Configuration for data janitor
+   */
+  public static final class DataJanitor {
+    public static final String PRUNE_ENABLE = "data.tx.prune.enable";
+    public static final String PRUNE_STATE_TABLE = "data.tx.prune.state.table";
+
+    public static final boolean DEFAULT_PRUNE_ENABLE = false;
+    public static final String DEFAULT_PRUNE_STATE_TABLE = "default:data_tx_janitor_state";
```

Comment:
Good point, I removed the namespace from the default value. It can always be overridden in the configuration